src/Calculator/App.xaml

```diff
@@ -1766,6 +1810,29 @@
                     </Setter.Value>
                 </Setter>
             </Style>
+
+            <Style x:Key="VariableTextBoxStyle" TargetType="TextBox">
+                <Setter Property="Margin" Value="1,0,0,0"/>
+                <Setter Property="Padding" Value="2,6,2,2"/>
+                <Setter Property="TextAlignment" Value="Center"/>
+                <Setter Property="Background" Value="Transparent"/>
+                <Setter Property="BorderThickness" Value="0,0,0,1"/>
+                <Setter Property="FontSize" Value="14"/>
+                <Setter Property="InputScope" Value="Number"/>
+                <Setter Property="MinWidth" Value="30"/>
+            </Style>
+
+            <Style x:Key="VariableTextErrorBoxStyle" TargetType="TextBox">
```

Comment:
Except the background and the borderbrush, this style is exactly the same than VariableTextBoxStyle.

What do you think about replacing it with:
```
<Style x:Key="VariableTextErrorBoxStyle"
                           BasedOn="{StaticResource VariableTextBoxStyle}"
                           TargetType="TextBox">
    <Setter Property="BorderBrush" Value="{ThemeResource GraphSettingsErrorBorderBrush}"/>
    <Setter Property="Background" Value="{ThemeResource GraphSettingsErrorBackgroundBrush}"/>
</Style>
```
?